src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -834,21 +837,26 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
         };
     }
 
-    private _generateNewHandTipMeshes() {
+    private _generateNewHandTipMeshes(disableNearInteraction: boolean) {
         // populate information for near hover, pick and pinch
-
-        const hoverIndexMeshTip = SphereBuilder.CreateSphere("IndexHoverSphere", { diameter: 1});
-        hoverIndexMeshTip.scaling.set(0.1, 0.1, 0.1);
-        hoverIndexMeshTip.isVisible = false;
-
-        const pickIndexMeshTip = SphereBuilder.CreateSphere("IndexPickSphere", { diameter: 1 });
-        pickIndexMeshTip.scaling.set(0.03, 0.03, 0.03);
-        pickIndexMeshTip.isVisible = false;
-
+        const sceneToRenderTo = this._options.useUtilityLayer ? this._options.customUtilityLayerScene || UtilityLayerRenderer.DefaultUtilityLayer.utilityLayerScene : this._scene;

```

Comment:
We are only supporting near interaction on **either** the main scene **or** the utility layer scene?

Reply:
sure, what would you like it to be. again, I used this from the way it is used in a different function in the same file :)

Reply:
Yeah, I've got no great suggestions. You don't have to change it, but if you did maybe something like sceneToCreateMeshOn or meshCreationScene.